src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Sln
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET modify solution file(s) command";
+
+        public const string AppDescription = "Command to add, remove and list projects from the solution (SLN) file.";
+
+        public const string AppHelpText = "Projects to add to the solution.";
+
+        public const string CmdSlnFile = "<SLN_FILE>";
+
+        public const string CmdSlnFileText = "Solution file to operate on. If not specified, the command will search the current directory for one.";
```

Comment:
Instead of here, can you add the string to CommonLocalizableStrings.cs? There is already a string called ArgumentsProjectOrSolutionDescription. I think you need to add 2 new ones: ArgumentsProjectDescription and ArgumentsSolutionDescription with the correct description. Basically before I had the argument named <PROJECT_OR_SOLUTION> because we didn't know if we were being invoked through dotnet add p2p or dotnet add proj. Now we can distinguish between the two.